packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page/entity-properties-block.tsx

```diff
@@ -1,4 +1,5 @@
 import { ThemeProvider } from "@emotion/react";
+/* eslint-disable-next-line -- no-restricted-imports */
```

Comment:
```suggestion
/* eslint-disable-next-line -- no-restricted-imports -- why? */
```

(optional)

Reply:
Removed the need for this since we can use our custom button in the file